<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YF Dataloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="tickerInput">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Ticker (Eg. QQQ, SCHD)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathInput">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter Saving path (default. C:)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectPathButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Download</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Status: 정상</string>
    </property>
   </widget>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
